Source/Core/VideoCommon/RenderBase.h

```diff
@@ -403,6 +406,8 @@ class Renderer
   void FinishFrameData();
 
   std::unique_ptr<NetPlayChatUI> m_netplay_chat_ui;
+
+  bool m_force_reload_textures = false;
```

Comment:
It looks like this is written from by the host thread in `ForceReloadTextures` and read by the video thread in `Swap` without any sort of fencing.
It should probably be an `atomic<bool>`.